src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -5489,8 +6179,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
     }
 
     void visit_StringConstant(const ASR::StringConstant_t &x) {
-        std::string s = unescape_string(al, x.m_s);
-        tmp = builder->CreateGlobalStringPtr(s);
+        tmp = builder->CreateGlobalStringPtr(x.m_s);
```

Comment:
To be reverted, I think. Can you please add this LFortran as well!

https://github.com/lcompilers/lpython/pull/1502/files